gazebo/igvc_control/launch/gazebo.launch

```diff
@@ -3,9 +3,12 @@
 <launch>
   <env name="GAZEBO_MODEL_PATH" value="$(find igvc_description)"/>
   <arg name="namespace" default="igvc"/>
+  <!-- Switch with commented line below for better performance -->
```

Comment:
I think here you mean uncomment below for terrain within the world

Reply:
Ya, I'll phase that better